src/cdk-lib/amazonaurora/aurora-default-vector-store.ts

```diff
@@ -156,7 +157,7 @@ export class AmazonAuroraDefaultVectorStore extends cdk.Resource {
       clusterIdentifier: this.clusterIdentifier,
       defaultDatabaseName: this.databaseName,
       vpc: this.vpc,
-      vpcSubnets: { subnetType: ec2.SubnetType.PUBLIC },
+      vpcSubnets: { subnetType: ec2.SubnetType.PRIVATE_ISOLATED },
```

Comment:
I also spotted this which is not super best practice. But as you realized, changing it makes the build fail. Also if we change it we probably would need to introduce VPC Endpoints for the RDS Data API used by the Lambda function.

Reply:
Running another build with the change above, resolving this for now